packages/crew_profile.rb

```diff
@@ -0,0 +1,48 @@
+require 'package'
+
+class Crew_profile < Package
+  description 'A profile for Chromebrew, to be used instead of ~/.*rc'
+  homepage 'https://github.com/skycocker/chromebrew'
+  version "#{CREW_VERSION}"
+  source_url 'https://github.com/skycocker/chromebrew/raw/511ee9cb3138e113df74ddf0dee057ef5d9a46fd/README.md'
+  source_sha256 '27b201cec82d903a1856972e6d7ff1ac58a67c761d729ecd7fd14f24fa9d9901'
+
+  depends_on "xdg_base"
+
+  def self.install
+    system "mkdir", "-p", "#{CREW_DEST_PREFIX}/etc"
+    system "mkdir", "-p", "#{CREW_DEST_HOME}"
+    system "touch", "$HOME/.profile" # To not overwrite already installed configuration
+    system "ln", "-s", "$HOME/.profile", "#{CREW_DEST_PREFIX}/etc/profile"
+  end
+
+  def self.postinstall
+    puts
+    puts "To create your very own Crew profile, please execute this:".lightblue
+    puts
+    puts "echo \"if [ -f #{CREW_PREFIX}/etc/profile ]; then\" >> ~/.bashrc".lightblue
+    puts "echo \"  # Source crew profile\" >> ~/.bashrc".lightblue
+    puts "echo \"  source #{CREW_PREFIX}/etc/profile\" >> ~/.bashrc".lightblue
+    puts "echo \"fi\" >> ~/.bashrc".lightblue
+    puts "source ~/.bashrc".lightblue
+    puts
+    puts "For Zsh, execute this:".lightblue
+    puts
+    puts "echo \"if [ -f #{CREW_PREFIX}/etc/profile ]; then\" >> ~/.zshrc".lightblue
+    puts "echo \"  # Source crew profile\" >> ~/.zshrc".lightblue
+    puts "echo \"  source #{CREW_PREFIX}/etc/profile\" >> ~/.zshrc".lightblue
+    puts "echo \"fi\" >> ~/.zshrc".lightblue
+    puts "source ~/.zshrc".lightblue
+    puts
+    puts "For Fish, execute this:".lightblue
+    puts
+    puts "echo \"if [ -f #{CREW_PREFIX}/etc/profile ]\" >> ~/.config/fish/config.fish".lightblue
```

Comment:
If fish isn't installed, this will puke, right?  Can we add another check to make sure `~/.config/fish/config.fish` exists before attempting to append?

Reply:
If fish isn't installed, why would they run that?

Reply:
Just saying.